### ISSUE TITLE
Cwd launch setting

### ISSUE THREAD

**Issue by JMPSequeira** [zero-plusplus/vscode-autohotkey-debug#173]
Would it be possible to support the current working directory (cwd) as a launch setting?

I request this as all of Ahk file crawling is relative to current working directory (1.1 at least).

Example:

- RootDir:
    - afolder\file.ahk
    - testfolder\test1.ahk
    - testfolder\test2.ahk

- Code on afolder\file.ahk:
 ```autohotkey
 Loop, Files, test*.ahk, R
     MsgBox, % A_LoopFileLongPath
 ```

Running from a command line on RootDir  `AutoHotkey .\afolder\file.ahk` will find the test files. 
But if debugging I can never have a different cwd thus never getting the expected results.

I already saw some discussion around this issue [here](https://github.com/zero-plusplus/vscode-autohotkey-debug/issues/52) but I do think the op didn't make a good case for it.

thanks for your time

**Comment by zero-plusplus** [comment]
**I am using the [DeepL translation](https://www.deepl.com/translator).**

Implemented `cwd` attribute in launch.json. This will be included in the next release.

Note that the specification of [A_WorkingDir](https://lexikos.github.io/v2/docs/Variables.htm#WorkingDir) has changed for v2, so you need to run the following code to make it the same as v1.
```ahk
SetWorkingDir(A_InitialWorkingDir)
```

The next release may take some time.

**Comment by JMPSequeira** [comment]
Thank you

**Comment by zero-plusplus** [comment]
@JMPSequeira
I released `1.10.0` which includes this feature.

Thanks for the great idea.
